components/formats-gpl/src/loci/formats/in/NDPISReader.java

```diff
@@ -25,31 +25,36 @@
 
 package loci.formats.in;
 
-import java.io.IOException;
-import java.util.ArrayList;
-
 import loci.common.DataTools;
 import loci.common.Location;
 import loci.common.RandomAccessInputStream;
-import loci.formats.ChannelSeparator;
-import loci.formats.CoreMetadata;
-import loci.formats.FormatException;
-import loci.formats.FormatReader;
-import loci.formats.FormatTools;
-import loci.formats.MetadataTools;
+import loci.formats.*;
 import loci.formats.meta.MetadataStore;
+import loci.formats.tiff.IFD;
+import ome.units.UNITS;
+import ome.units.quantity.Length;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+
 
 /**
  * NDPISReader is the file format reader for Hamamatsu .ndpis files.
  *
  * @author Melissa Linkert melissa at glencoesoftware.com
+ * @author Manuel Stritt (manuel.stritt at actelion.com)
  */
 public class NDPISReader extends FormatReader {
 
   // -- Fields --
 
   private String[] ndpiFiles;
-  private ChannelSeparator[] readers;
+  private NDPIReader[] readers;
+  private int[] bandUsed;
+  private int[] samplesPerPixel;
+  private final static int TAG_CHANNEL = 65434;
+  private final static int TAG_EMISSION_WAVELENGTH = 65451;
```

Comment:
Thanks for the clarification. Including this change in a minor version increment certainly makes sense.